enterprise/frontend/src/embedding-sdk/cli/constants/database.ts

```diff
@@ -0,0 +1,43 @@
+/** Show popular database engines in the CLI */
+export const CLI_SHOWN_DB_ENGINES = [
```

Comment:
This is determined by https://stats.metabase.com/question/18990-popularity-of-db-engines-for-data-sources-based-on-cloud from stats